era5_job.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#SBATCH -J gfs_process
+#SBATCH -o /contrib/Sadegh.Tabas/ML4BC/era5_output.out
```

Comment:
Is this job card working on aws cloud?

Reply:
I removed the personal directories. Yes if you look at /contrib/Sadegh.Tabas/ML4BC/ I did run the code (still running) and preparing the training data. Thanks